src/currencies/sanitizeValueString.js

```diff
@@ -18,6 +18,8 @@ export const sanitizeValueString = (
   let display = "";
   let value = "";
   let decimals = -1;
+  const trimLeadingZeros = new RegExp(`^([0]+)(?!$)(?!\\${dot})`, "g");
```

Comment:
can we try to implement this in the `for` loop under it? it's really designed to be as efficient as possible because it's a critical part to be efficient, especially for mobile (otherwise the keyboard input does not answer fast enough and it's going jumpy)

Reply:
makes sense, we don't really need this regexp anyway